lib/getPackageReportData.js

```diff
@@ -53,7 +53,7 @@ function getPackageReportData(packageEntry, callback) {
 
 		var author = json.author
 		if (_.isObject(author)) {
-			author = author.name;
+			author = author.name || author.email || ""
 			if (author.email) { author = author + ' ' + author.email; }
```

Comment:
And then we have the next problem here: if `author.name` doesn't exist, the we do not have to add the space character (` + ' '`).
So perhaps we should change the line to something like:
```suggestion
			if (author.email) { 
			  if (author.length > 0) {
			    author += ' ';
			  }
			  author += author.email;
			}
```